### PR TITLE
Add rankprofile to log message.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp [hunk 1]
```diff
@@ -311,7 +313,8 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
         std::lock_guard<std::mutex> guard(_statsLock);
         _stats.add(my_stats);
         if (my_stats.softDoomed()) {
-            LOG(info, "Triggered softtimeout limit=%1.3f and duration=%1.3f", softLimit.sec(), duration.sec());
+            LOG(info, "Triggered softtimeout limit=%1.3f and duration=%1.3f, rankprofile=%s",
```
Comment (geirst) [inline searchcore/src/vespa/searchcore/proton/matching/matcher.cpp]: Consider change wording to avoid confusion that softtimeout is actually happening.

**Review by geirst — APPROVED** [review]
Looks good, only a nitpick comment

**Comment by baldersheim** [comment]
A bit better...

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**